lib/order.rb

```diff
@@ -0,0 +1,40 @@
+require_relative 'menu'
+require_relative 'messager'
+
+class Order
+
+  attr_reader :list, :menu, :order, :total_cost, :message, :messager
+
+  def initialize(menu = Menu.new, messager = Messager.new)
+    @list = []
+    @menu = menu
+    @messager = messager
+  end
+
+  def read_menu
+    @menu.display_menu
+  end
+
+  def choose(item)
+   @choice = @menu.set_menu.select { |key, value| item.include? key }
+     "You have added 1x #{item} to your order."
```

Comment:
Lint/Void: Literal `"You have added 1x #{item} to your order."` used in void context.